cmd/osctl/cmd/cluster.go

```diff
@@ -84,29 +93,40 @@ func create(ctx context.Context) (err error) {
 	}
 
 	memory := int64(clusterMemory) * 1024 * 1024
+	diskSize := int64(clusterDiskSize) * 1024 * 1024
 
 	// Validate CIDR range and allocate IPs
-	fmt.Println("validating CIDR and reserving master IPs")
+	fmt.Println("validating CIDR and reserving IPs")
 
 	_, cidr, err := net.ParseCIDR(networkCIDR)
 	if err != nil {
 		return fmt.Errorf("error validating cidr block: %w", err)
 	}
 
+	// Host addr at 1st IP in range, ex. 192.168.0.1
+	var hostIP net.IP
+
+	hostIP, err = talosnet.NthIPInNetwork(cidr, 1)
+	if err != nil {
+		return err
+	}
+
 	// Set starting ip at 2nd ip in range, ex: 192.168.0.2
-	ips := make([]string, masters)
+	ips := make([]net.IP, masters+workers)
```

Comment:
firecracker provisioner doesn't have any way to allocate IPs on its own, so I went ahead and allocated all the IPs in the `osctl` for both docker & firecracker